application-services/custom/camera-management/README.md

```diff
@@ -1,340 +1,13 @@
-    level=INFO ts=2022-07-11T22:26:11.581149638Z app=app-camera-management source=evam.go:115 msg="View inference results at 'rtsp://<SYSTEM_IP_ADDRESS>:8555/<device name>'"
-    ```
-
-1. Use the URI from the log to view the camera footage with analytics overlayed.
-    ```bash
-    ffplay 'rtsp://<SYSTEM_IP_ADDRESS>:8555/<device name>'
-    ```
-
-   Example Output:  
-   ![example analytics](./images/example-analytics.png)  
-
-1. Press the red square stop button to shut down the pipeline.
-
-
-### API Log
-
-The API log shows the status of the 5 most recent calls and commands that the management has made. This includes important information from the responses, including camera information or error messages.
-
-![API Logs](./images/api-log.png)  
-
-1. Expand a log item to see the response
-
-   Good response:
-   ![good api response](./images/good-response.png)  
-   Bad response:
-   ![bad api response](./images/bad-response.png)  
-
-### Inference Events
-
-![inference events default](./images/inference-events-default.png)  
-
-1. To view the inference events in a json format, click the `Stream Events` button.
-
-![inference events](./images/inference-events.png)  
-
-### Inference results in Edgex
-
-To view inference results in Edgex, open Edgex UI [http://localhost:4000](http://localhost:4000), click on the `DataCenter`
-tab and view data streaming under `Event Data Stream`by clicking on the `Start` button.
-
-![inference events](./images/inference-edgex.png)
-
-### Next steps
-A custom app service can be used to analyze this inference data and take action based on the analysis.
-
-## Additional Development
-
-> **Warning**: The following steps are only useful for developers who wish to make modifications to the code
-> and the Web-UI.
-
-#### Development and Testing of UI
-##### 1. Build the production web-ui
-This builds the web ui into the `web-ui/dist` folder, which is what is served by the app service on port 59750.
-```shell
-make web-ui
-```
-
-##### 2. Serve the Web-UI in hot-reload mode
-This will serve the web ui in hot reload mode on port 4200 which will recompile and update anytime you make changes to a file. It is useful for
-rapidly testing changes to the UI.
-```shell
-make serve-ui
-```
-
-Open your browser to [http://localhost:4200](http://localhost:4200)
+> The **main** branch of this repository contains work-in-progress development code for the upcoming release, and is **not guaranteed to be stable or working**.
+> It is only compatible with the [main branch of edgex-compose](https://github.com/edgexfoundry/edgex-compose) which uses the Docker images built from the **main** branch of this repo and other repos.
+>
+> **The source for the latest release can be found at [Releases](https://github.com/edgexfoundry/edgex-examples/releases).**
 
 
+## General
+Visit the [EdgeX Documentation page](https://docs.edgexfoundry.org/3.0/examples/AppServiceExamples/Ch-CameraManagement) to learn more about the latest release.
```

Comment:
In think this might be better?
```suggestion
Visit the [EdgeX Documentation page](https://docs.edgexfoundry.org/3.0/examples/AppServiceExamples/Ch-CameraManagement) to learn more about the latest camera management example app
```